### PR TITLE
Sync main with next

### PR DESCRIPTION
**Pull request by pieckenst**

### CODE REVIEW

**Comment by pieckenst** [comment]
@electricduck just main to next sync

File: src/common/core.yaml [hunk 1]
```diff
@@ -16,13 +16,17 @@ repos:
  - fedora-36-updates
  - fedora
  - fedora-updates
+ #- terra # f37+ only
```
Comment (electricduck) [inline src/common/core.yaml:19]: This should be uncommented as next is f38

File: src/packages/pantheon-apps.yaml [hunk 2]
```diff
@@ -1,15 +1,13 @@
 packages:
- - com.github.stsdc.monitor
  #- elementary-calculator
  #- elementary-calendar
  #- elementary-camera
  #- elementary-capnet-assist
  - elementary-code
- - elementary-mail
+ #- elementary-mail
```
Comment (electricduck) [inline src/packages/pantheon-apps.yaml:7]: This package doesn't exist in Terra and needs to be commented out

File: src/packages/pantheon-apps.yaml [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
  - elementary-music
  - elementary-photos
  #- elementary-screenshot-tool
  - elementary-sideload
- - elementary-tasks
+ #- elementary-tasks
```
Comment (electricduck) [inline src/packages/pantheon-apps.yaml:12]: This package doesn't exist in Terra and needs to be commented out

File: src/packages/pantheon.yaml [hunk 4]
```diff
@@ -41,9 +41,9 @@ packages:
...
  - switchboard-plug-onlineaccounts
  - switchboard-plug-pantheon-shell
- - switchboard-plug-parental-controls # Untested
+ - switchboard-plug-parental-controls
  - switchboard-plug-printers
- - switchboard-plug-power # Limited functionality
+ - switchboard-plug-power
  - switchboard-plug-security-privacy
  - switchboard-plug-sharing
  - switchboard-plug-sound
```
Comment (electricduck) [inline src/packages/pantheon.yaml:49]: `switchboard-plug-locale` and `switchboard-plug-useraccounts` are currently not on Terra, so these need commenting out

**Review by electricduck — CHANGES_REQUESTED** [review]

File: src/packages/pantheon.yaml (hunk 4 above)
Comment (pieckenst) [reply src/packages/pantheon.yaml:49]: Edited it

File: src/common/core.yaml (hunk 1 above)
Comment (pieckenst) [reply src/common/core.yaml]: done:)

File: src/packages/pantheon-apps.yaml (hunk 3 above)
Comment (pieckenst) [reply src/packages/pantheon-apps.yaml:12]: - shouldn't that be covered by the changes you already did? looks to be fine https://github.com/pieckenst/sodalite/blob/release/next/src/packages/pantheon-apps.yaml ?

**Pull request merged**